crates/cdk/Cargo.toml

```diff
@@ -55,16 +55,16 @@ thiserror = "1"
 futures = { version = "0.3.28", default-features = false, optional = true, features = [
     "alloc",
 ] }
-url = "2.3"
+url = "2.5"
 utoipa = { version = "4", optional = true }
 uuid = { version = "1", features = ["v4", "serde"] }
 
 # -Z minimal-versions
-sync_wrapper = "0.1.2"
-bech32 = "0.9.1"
+sync_wrapper = "1"
+bech32 = "0.11"
```

Comment:
These are not direct deps of ours but transitive dependencies that do not have the minimal version defined correctly so they need to be defined here. So we cant change to a new major or minor version as we want to set the correct major/minor version for the dep. 

I haven't run the minimal check in a bit so I should redo it and add to CI and maybe we will be able to update or remove these if your direct dep has been updated. https://github.com/cashubtc/cdk/issues/505